### PR TITLE
feat: implement upload from files

### PR DESCRIPTION
**Pull request by ArthurChalard**
Implement the logic for the button "Import from local files" in the document list.

When the user click on the button, a file picker is shown and the user can select any pdf, jpeg or png file from his local files. When he has selected it, the file is sent to the server.

### CODE REVIEW

**Comment by ArthurChalard** [comment]
Fix #116

File: app/src/main/java/com/github/se/travelpouch/model/documents/DocumentViewModel.kt
Comment (appellet) [inline app/src/main/java/com/github/se/travelpouch/model/documents/DocumentViewModel.kt:1]: The documentation would be appreciated to understand the function and to remain consistent with the rest of the project.

File: app/src/main/java/com/github/se/travelpouch/ui/documents/DocumentList.kt
Comment (appellet) [inline app/src/main/java/com/github/se/travelpouch/ui/documents/DocumentList.kt:1]: As said in the other file, you can add some documentation to class and function.

**Review by appellet — CHANGES_REQUESTED** [review]
Great change. I have no complaints. Only small modification would be on the documentation in order to align with the conventions of our project.

**Review by appellet — APPROVED** [review]

**Pull request merged**